src/event.c

```diff
@@ -133,6 +134,7 @@ void xmpp_run_once(xmpp_ctx_t *ctx, unsigned long timeout)
                 break; /* partial write or an error */
 
             /* all data for this queue item written, delete and move on */
+            xmpp_debug(ctx, "xmpp", "Finished writing queue (%p).", sq);
```

Comment:
Please, remove this debug message. At this point the queue is not finished, this is just an iteration. This debug output will be excessive.

Reply:
Hmm, one could argue that this is the correct place for the `SENT: ...` debug message, or not?